src/app/learn/reference/cli-commands/certs/index.html

```diff
@@ -0,0 +1,12 @@
+<stache
+  pageTitle="certs">
+
+  <p>
+    <sky-code>skyux certs</sky-code> manages the local certificate necessary to securely run <sky-code>skyux serve</sky-code> and <sky-code>skyux build --serve</sky-code>.
+  </p>
+
+  <stache-page-anchor>
+    Options
```

Comment:
Was this header an oversight? Or did you set up the header because you still need to add entries for flags with this command?